server/datastore/mysql/invites_test.go

```diff
@@ -12,10 +12,29 @@ import (
 	"gopkg.in/guregu/null.v3"
 )
 
-func TestCreateInvite(t *testing.T) {
+func TestInvites(t *testing.T) {
 	ds := CreateMySQLDS(t)
-	defer ds.Close()
 
+	cases := []struct {
+		name string
+		fn   func(t *testing.T, ds *Datastore)
+	}{
+		{"Create", testInvitesCreate},
+		{"List", testInvitesList},
+		{"Delete", testInvitesDelete},
+		{"ByToken", testInvitesByToken},
+		{"ByEmail", testInvitesByEmail},
+		{"Invite", testInvitesInvite},
+	}
+	for _, c := range cases {
+		t.Run(c.name, func(t *testing.T) {
+			defer TruncateTables(t, ds, "teams", "invites", "invite_teams")
```

Comment:
Would truncating all the tables be much slower?

Reply:
Probably not, though some are seeded by the sql schema file and should be left alone. It's usually quite clear when one is missing (as a subsequent test will fail with a duplicate error or wrong number of elements, etc.), but yeah I don't think it'd be significantly slower. I'll try it out.

Reply:
Updated the PR, it seems roughly as fast (10-12 seconds on my machine).